client/js/test/table-view.test.js

```diff
@@ -13,6 +13,35 @@ describe('table-view', () => {
 		document.documentElement.innerHTML = html;
 	});
 
+	describe('table footer', () => {
+		it('it has right size', () => {
```

Comment:
Test assertions should also describe the intended functionality. Here, what is "the right size"? Is that number of columns or rows? Cell dimensions? Be specific.

Also, test assertions read like sentences, so start them with a verb. Here, yours would say "it it has the right size".